gdal/swig/python/gdal-utils/scripts/batch_creator.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# ******************************************************************************
+#
+#  Project:  GDAL
+#  Purpose:  Create Windows batch wrappers to run the python scripts
```

Comment:
this script doesn't seem to be used by anything else in this PR ?

Reply:
Right. 
Either a Windows user can run it once to create the batch files or I would need to find a way to run it during instalation of the wheel on Windows. 
What do you think?

Reply:
The later would probably be ideal. Otherwise the existence of this script should be mentioned in the documentation of the package

Reply:
OK, Do you happen to know how to get the path of the scripts folder (i.e. `C:\Python38\Scripts`) so I can run it as post-install ?

Reply:
It seems that [it is not possible to run post install scripts with wheel distributions.](https://www.javaer101.com/en/article/34927268.html) 
So we either need to distribute the utils as sdist (which I prefer not to) and use [post install](https://stackoverflow.com/questions/20288711/post-install-script-with-python-setuptools)
Or to install the batches the same way we install the scripts, but then they will be installed for all OSs (they would be useless on other OSs but not harmful I guess)
I don't think it justifies to make a separate wheel for windows only...
Or to advise the user in the docs to run the batch creator if they wish to do so.

What do you think?

Reply:
I've documented it properly in the README:
In case you want to run GDAL scripts on Windows,
If don't have `python.exe` associated with py file extension
(For instance, because you associate them with your IDE)
you might want to create batch wrappers for the gdal python scripts,
Otherwise you won't be able to run the scripts easily.
To create the batch files, Run the following command (after installing `gdal-utils`):

`python -m osgeo_utils.auxiliary.batch_creator`